From 0524bca92ee64c404dde427a5f11f94e37badf0b Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 21:26:13 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the direct GnuPG verification with %openpgpverify, which
uses Sequoia sqv, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh victortoso-E37A484F.keyring from keyservers/WKD: same
certificate(s), with self-signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 mingw-usbredir.spec | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/mingw-usbredir.spec b/mingw-usbredir.spec
index 0a92f12..50ce687 100644
--- a/mingw-usbredir.spec
+++ b/mingw-usbredir.spec
@@ -2,7 +2,7 @@
 
 Name:           mingw-usbredir
 Version:        0.14.0
-Release:        6%{?dist}
+Release:        7%{?dist}
 Summary:        MinGW USB network redirection protocol libraries
 
 # Automatically converted from old format: LGPLv2+ - review is highly recommended.
@@ -25,6 +25,7 @@ BuildRequires:  mingw64-glib2
 BuildRequires:  git-core
 BuildRequires:  meson
 BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 
 %description
 The usbredir libraries allow USB devices to be used on remote and/or virtual
@@ -76,7 +77,7 @@ applications that use usbredir with MinGW.
 
 
 %prep
-gpgv2 --quiet --keyring %{SOURCE2} %{SOURCE1} %{SOURCE0}
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -S git_am -p1 -n usbredir-%{version}
 
 %build
@@ -135,6 +136,9 @@ find $RPM_BUILD_ROOT -name "*.la" -delete
 
 
 %changelog
+* Mon Oct 26 2026 Jakub Jelen <jjelen@redhat.com> - 0.14.0-7
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 0.14.0-6
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.56.0

